src/app/pages/LendingPage/components/CurrencyContainer/PoolChart.tsx

```diff
@@ -0,0 +1,312 @@
+import React from 'react';
+
+import ComparisonChart from 'app/components/FinanceV2Components/ComparisonChart';
+import { getAssetColor } from 'app/components/FinanceV2Components/utils/getAssetColor';
+import { LendingPool } from 'utils/models/lending-pool';
+
+interface Props {
+  pool: LendingPool;
+}
+
+/* Sample data */
+const testData: any[] = [
```

Comment:
There is APR data in the backend (old lending) which can be used to show at least one of the lines i think.